### ISSUE TITLE
NO way to delete calendar

### ISSUE THREAD

**Issue by Gabriele69** [ubports/calendar-app#15]
If I go to the calendars section, I can only add new calendars (personal, google calendar, holidays, etc.) but I can not delete the calendars

g

**Comment by GermoVeltmaat** [comment]
You can remove (caldav) accounts through the 'general settings':'accounts'

**Comment by hummlbach** [comment]
But theres also no way to add local calendars.(?)

**Comment by hummlbach** [comment]
Suggestion (for calendar management view):
- use bottom edge to add calendars (local as well as remote)
- swipe right on calendar shows trash icon to remove calendar

**Comment by Gabriele69** [comment]
No. On my bq Aquaris e4.5 (RC 15.04 r15) isn't possible to do swipe. It's possible only select or not select the calendars (Or add a new calendar).

**Comment by hummlbach** [comment]
That wasn't a suggestion for how you should use it, but for how it could be implemented... ;)

**Comment by ntorquet** [comment]
It will be a great improvement to can delete a calendar. In my case, I changed several times the server where I host my 10 calendars. Now I have more than 30 calendars... It will be very nice to clean a bit the old ones!
This app is very useful now, thanks to good improvements. It is very well design, I really like it! Many thanks to the contributors!!!
